app/code/Magento/CatalogGraphQl/Model/Category/CategoryFilter.php

```diff
@@ -71,6 +72,7 @@ public function getResult(array $criteria, StoreInterface $store)
         $categoryIds = [];
         $criteria[Filter::ARGUMENT_NAME] = $this->formatMatchFilters($criteria['filters'], $store);
         $criteria[Filter::ARGUMENT_NAME][CategoryInterface::KEY_IS_ACTIVE] = ['eq' => 1];
+        $criteria[Sort::ARGUMENT_NAME][CategoryInterface::KEY_POSITION] = ['ASC'];
```

Comment:
@Dnyomo can you please verify and confirm that for categories (e.g in menu) also have a sort order by position ASC ?